### ISSUE TITLE
GSLB 'depends-on' directive missing on pool member

### ISSUE THREAD

**Issue by mattrobinson1** [F5Networks/f5-appsvcs-extension#329]
AS3 doesn't offer configuration of the 'depends-on' directive for A or AAAA pool members on a GSLB config. This is a heavily used feature in a configuration I'm working on and finding workarounds are pretty painful.

Below I have listed the relevant section from tmsh docs showing the depends-on params and also the options available for a A record pool member in GSLB via AS3 today. Full support for depends-on in this context would *really* helpful right now.

### TMSH for GSLB pool member:
```
members
      [ add | delete | modify | replace-all-with ] {
        [server-name:vs-name] {
         options:
           app-service [[string] | none]
           depends-on none
           depends-on
             [ add | delete | modify | replace-all-with ] {
               [server-name:vs-name]...
           }
           description [string]
           [disabled | enabled]
          ...
       }...
    }
```

### AS3 for GSLB pool member (type A as an example):
```
GSLB_Pool_Member_A

Declares member of the GSLB pool
Properties:

Name (Type)
- label (string)	
- ratio (integer)	
- remark (string)	
- server (Pointer_GSLB_Server)
- virtualServer (string) 

```

**Comment by dstokesf5** [comment]
Thank you for submitting this feature request. I have added it to our internal product backlog as AUTOTOOL-1893.

**Comment by dstokesf5** [comment]
This was resolved in the 3.24.0 release.
